app/src/main/java/woowacourse/shopping/feature/list/item/ProductView.kt

```diff
@@ -1,20 +1,37 @@
 package woowacourse.shopping.feature.list.item
 
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
 sealed interface ProductView {
 
-    data class ProductItem(
+    @Parcelize
+    data class CartProductItem(
         val id: Int,
         val imageUrl: String,
         val name: String,
         val price: Int,
-    ) : ProductView
+    ) : ProductView, Parcelable {
+        var count: Int = DEFAULT_COUNT
+            private set
+
+        fun updateCount(count: Int): CartProductItem {
+            this.count = count
+            return this
+        }
+
+        companion object {
+            const val NO_COUNT = 0
+            const val DEFAULT_COUNT = 1
```

Comment:
```suggestion
            private const val DEFAULT_COUNT = 1
```
클래스 내부에서만 사용하는 변수라면 private 접근자를 붙여볼까요?

Reply:
반영하였습니다!